torch/csrc/autograd/python_cpp_function.cpp

```diff
@@ -77,6 +77,14 @@ PyObject* THPCppFunction_call(
 
 int THPCppFunction_traverse(PyObject* self, visitproc visit, void* arg) {
   auto& fn = *((THPCppFunction*)self)->cdata;
+  if ((((THPCppFunction*)self)->cdata).use_count() > 1) {
```

Comment:
This definitely makes the diff easier to read but I think it's better to guard the calls below rather than early returning.